akka-distributed-data/src/main/protobuf/ReplicatedDataMessages.proto

```diff
@@ -72,7 +72,29 @@ message ORMap {
   }
   
   required ORSet keys = 1;
-  repeated Entry entries = 2;  
+  repeated Entry entries = 2;
+}
+
+message ORMapDeltaGroup {
+  message Entry {
+    required ORMapDeltaOp operation = 1;
+    required ORSet underlying = 2;
+    required sint32 zeroTag = 3;
+    optional string stringKey = 4;
+    optional OtherMessage value = 5;
+    optional sint32 intKey = 6;
+    optional sint64 longKey = 7;
+    optional OtherMessage otherKey = 8;
```

Comment:
That is not exactly true, `ORMultiMap.Entry` has also an `ORSet` which is not needed here. In addition to that - we are now having up to one `entryData` but later, for combining `updateDeltaOp` s we will need to have proper map here, but still only one `ORSet`, so the `ORMultiMap.Entry` cannot be re-used. But I have re-factored the common part into `ORMapDeltaGroup.MapEntry` which is now `optional` but later can be made `repeated`.